web/react/components/channel_header.jsx

```diff
@@ -194,6 +194,17 @@ export default class ChannelHeader extends React.Component {
             );
         }
 
+        const popoverListMembers = [];
+        if (!isDirect) {
+            popoverListMembers.push(
+                <PopoverListMembers
+                    members={this.state.users}
+                    memberCount={this.state.userCount}
+                    channelId={channel.id}
+                />
```

Comment:
Nope I do `let variableName;` all the time. React treats undefined as the same as null (which I believe is pretty common in javascript)